naplib/naplab/process_ieeg.py

```diff
@@ -50,7 +50,7 @@ def process_ieeg(
     """
     data_path : str path-like
         String specifying data directory (for TDT) or file path for raw data file
-    alignment_dir : str path-like
+    stim_dir : str path-like
```

Comment:
Also, can you change the docstring for stim_order_path to say "defaults to ``alignment_dir``, and also to say "correspond to the names of the .wav files in ``alignment_dir``?